packages/odyssey-react/src/components/Status/Status.tsx

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ComponentProps, ComponentPropsWithoutRef } from "react";
+import { forwardRef } from "react";
+import { useCx, withStyles } from "../../utils";
+import styles from "./Status.module.scss";
+
+import ScreenReaderText from "../ScreenReaderText";
+
+export type StatusVariants = "neutral" | "success" | "caution" | "danger";
+
+interface Props
+  extends Omit<ComponentPropsWithoutRef<"div">, "style" | "className"> {
```

Comment:
consider adding `role` to the omitted string union here as this prop is no available to be set (we hardcode it to `status` below). Actually looking at this prop definition we'll need to leverage `useOmit` and spread the `{...omitProps}` into the `div` element to make our implementation align with our typed props!